tf/modules/ooni_dev_user/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "ooni_dev_user_arn" {
+  value = aws_iam_user.ooni_dev_user.arn
+}
+
+output "ooni_dev_user_access_key" {
+    value = aws_iam_access_key.ooni_dev_user.encrypted_secret
```

Comment:
You should mark this output as `sensitive = true` that it's not printed. I also think that since you are storing it in the secrets manager there is no need to generate it as an output, since all we need is the ability to read it from the secrets store, but we don't care to have the actual value be used as an input to other things.